openpype/hosts/max/plugins/create/create_render.py

```diff
@@ -30,3 +30,28 @@ def create(self, subset_name, instance_data, pre_create_data):
             RenderSettings(self.project_settings).set_render_camera(sel_obj)
         # set output paths for rendering(mandatory for deadline)
         RenderSettings().render_output(container_name)
+
+    def get_instance_attr_defs(self):
+        ocio_display_view_transform_list = ["sRGB||ACES 1.0 SDR-video"]
+        if int(get_max_version()) >= 2024:
+            display_view_default = ""
+            ocio_display_view_transform_list = []
+            colorspace_mgr = rt.ColorPipelineMgr
+            displays = colorspace_mgr.GetDisplayList()
+            for display in sorted(displays):
+                views = colorspace_mgr.GetViewList(display)
+                for view in sorted(views):
+                    ocio_display_view_transform_list.append({
+                        "value": "||".join((display, view))
+                    })
+                    if display == "ACES" and view == "sRGB":
+                        display_view_default = "{0}||{1}".format(
+                            display, view
+                        )
```

Comment:
Note that `get_instance_attr_defs` is called PER instance and quite often, more often that `apply_settings`. You could potentially optimize this by caching the result in apply settings to a class attribute.